explorer/src/Pos/Explorer/Web/Error.hs

```diff
@@ -2,17 +2,21 @@
 
 module Pos.Explorer.Web.Error
        ( ExplorerError (..)
+       , _Internal
        ) where
 
+import           Control.Lens        (makePrisms)
 import qualified Data.Text.Buildable
 import           Formatting          (bprint, stext, (%))
 import           Universum
 
-data ExplorerError =
+newtype ExplorerError =
     -- | Some internal error.
-    Internal !Text
+    Internal Text
```

Comment:
If it's going to be used through `Lens`es, maybe put an underscore here and import it without the underscore.

Reply:
As for me, it looks like departing from the usual workflow ; -)

Reply:
I don't know the rules, but with `WalletError` (and others) we are doing the same thing (w/o underscore for data constructors) https://github.com/input-output-hk/cardano-sl/blob/master/lib/src/Pos/Wallet/Web/Error/Types.hs#L16-L27